app/models/manageiq/providers/kubernetes/container_manager/metrics_capture.rb

```diff
@@ -1,10 +1,57 @@
+require 'hawkularclient'
+
 module ManageIQ::Providers
   class Kubernetes::ContainerManager::MetricsCapture < BaseManager::MetricsCapture
+    require_nested :CaptureContext
+
+    INTERVAL   = 20 # seconds
+
+    VIM_STYLE_COUNTERS = {
```

Comment:
It is a naming convention, probably @chessbyte or @blomquisg know the exact origin. Anyway it's the constant name used in all `metrics_capture.rb`.